conversions/octal_to_decimal.py

```diff
@@ -9,26 +9,40 @@ def oct_to_decimal(oct_string: str) -> int:
     >>> oct_to_decimal("-45")
     -37
     >>> oct_to_decimal("2-0Fm")
+    Traceback (most recent call last):
+    ...
     ValueError: Non-octal value was passed to the function
     >>> oct_to_decimal("")
+    Traceback (most recent call last):
+    ...
     ValueError: Empty string value was passed to the function
     >>> oct_to_decimal("19")
+    Traceback (most recent call last):
+    ...
     ValueError: Non-octal value was passed to the function
     """
+    # Strip oct_string of whitespaces
```

Comment:
This comment repeats what the code says.  Comments should not be about _what_ the code is doing, but should be about _why_ we are doing something.  Why are we removing whitespace?  Why are we removing the minus sign.

Reply:
Ok